main.go

```diff
@@ -212,6 +212,9 @@ available string filters: event, set, uts, comm
 Boolean filters use the following operator: '!'
 available boolean filters: container
 
+Event argument filters are used to filter a specific event by its arguments.
+The following format should be used: "event_name:event_arg=val" or "event_name:event_arg!=val"
```

Comment:
don't you think `.` is more intuitive than `:` when specifying lookup?

Reply:
Good idea. I'll do that.
This also made me find a small bug.
I checked if we don't have an event or argument with a dot in it, and I found the following:
{Type: "struct sched_param*", Name: "external.ArgMeta"}},
For 3 different system calls: SchedSetparamEventID, SchedGetparamEventID, SchedSetschedulerEventID
Probably happened during renaming of param to external.ArgMeta